components/ProgramsTitleDescription.tsx

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import { Box, Typography } from '@mui/material';
+
+interface ProgramsTitleDescriptionProps {
+    title: string;
+    description: string;
+    marginTop: number;
+    marginBottom: number;
+    color: string // uses colours from globals.css
+  }
+
+export default function ProgramsTitleDescription({title, description, marginTop, marginBottom, color}: ProgramsTitleDescriptionProps) {
+  return (
+    <Box sx={{ mt: marginTop, mb: marginBottom }}>
+      <Typography className={color} variant='h4' sx={{ mb: 1 }}>
```

Comment:
make sure you're using levels instead of variant so our fonts are consistent! I think for the title, it should be subtitle-lg, and then for the description, it should just be body

Reply:
if I do this - it centers the heading which looks really funny, what can I do instead to use 'subtitle-lg' but not have it centered